scripts/commands/entityanimation.lua

```diff
@@ -0,0 +1,40 @@
+---------------------------------------------------------------------------------------------------
+-- func: entityanimation
```

Comment:
so turns out we have commands for this already, but they are type specific - one for players one for non players.

https://github.com/LandSandBoat/server/blob/base/scripts/commands/animatenpc.lua#L53

Reply:
Removed command entirely